capstone/pom.xml

```diff
@@ -40,7 +40,7 @@
     <dependency>
       <groupId>com.google.appengine</groupId>
       <artifactId>appengine-api-stubs</artifactId>
-      <version>1.9.59</version>
+      <version>1.9.64</version>
```

Comment:
I imagine we need to make this change to also use the remote api or is it just for being updated?

Reply:
I updated it to this because the simple remote API example I was following used this version so I figured it was probably best to update. Does that answer your question?